### PR TITLE
fix: change ui of booking fields in booking page

### LINKED ISSUE #7748: [CAL-1263] booking questions (formerly additional inputs) need full width not coluns
for all additional inputs (now called booking questions) we should make it full width and not columns

now:

should be:

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1263](https://linear.app/calcom/issue/CAL-1263/booking-questions-formerly-additional-inputs-need-full-width-not)</sub>

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

Fixes https://github.com/calcom/cal.com/issues/7748

AFTER:-

Screenshot 2023-03-16 at 7 46 56 PM

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by Udit-takkar** [comment]
@Jaibles any other suggestion you have in mind ?

File: apps/web/pages/booking/[uid].tsx [hunk 1]
```diff
@@ -563,7 +563,8 @@ export default function Success(props: SuccessProps) {
                         </div>
                       </>
                     )}
-
+                  </div>
+                  <div className="text-bookingdark dark:border-darkgray-200 mt-8 text-left dark:text-gray-300">
```
Comment (sean-brydon) [inline apps/web/pages/booking/[uid].tsx:567]: I dont think booking dark as a light mode value 🤔 as its the contrast value caluclated for the darkmode as a background 

In theory i can set darkmode to a colour that constrast value is white and this text would be very hard to read. 

CleanShot 2023-03-21 at 23 11 37@2x

```suggestion
                  <div className="text-gray-300 dark:border-darkgray-200 mt-8 text-left dark:text-darkgray-300">
```

**Review by sean-brydon — CHANGES_REQUESTED** [review]

File: apps/web/pages/booking/[uid].tsx (hunk 1 above)
Comment (Udit-takkar) [reply apps/web/pages/booking/[uid].tsx:567]: @sean-brydon text-bookingdark is used everywhere in /booking/[uid] page

**Review by sean-brydon — APPROVED** [review]
Think i got confused here 

Sorry! Nice job.

**Review by PeerRich — APPROVED** [review]

**Pull request merged**